### PR TITLE
Canonicals are wrong

### LINKED ISSUE #1836: Canonicals are wrong
**Describe the bug**
On the current live article on .co.uk, the canonical URL is .co.uk.  This is wrong.  The canonical should always be .com.

**To Reproduce**

1. Go to https://www.bbc.co.uk/news/articles/c744mjrym37o
2. View source
3. Search for "canonical"
4. Observe that it says .co.uk

**Expected behavior**
Canonical URL should always be .com

### PR DESCRIPTION
**Pull request by rhenshaw56**
Resolves #1836

Changed canonical link from bbc.co.uk to bbc.com

**Code changes:**

- changed origin in getOriginContext.test.js to bbc.com and updated logic accordingly
-  updated tests and snapshots

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

**Review by 12 — COMMENTED** [review]
Should this not be `.co.uk`?
```html
<link data-react-helmet="true" rel="alternate" href="https://www.bbc.com/news/articles/c6v11qzyv8po" hrefLang="en"/>
```

File: src/app/contexts/RequestContext/getOriginContext.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const getOriginContext = bbcOrigin => {
-  let origin = 'https://www.bbc.co.uk';
-  let isUK = true;
+  let origin = 'https://www.bbc.com';
```
Comment (dr3) [inline src/app/contexts/RequestContext/getOriginContext.js]: Setting .com as the default has huge affects in the application, we default to .co.uk to ensure that if the application is missing something, we dont accidentally show a UK user something like adverts. Chatting to @sareh this doesn't feel like something we should do to fix a metadata issue. Can you catch up with her please?

**Review by dr3 — CHANGES_REQUESTED** [review]

**Comment by rhenshaw56** [comment]
> Should this not be `.co.uk`?
> 
> ```
> <link data-react-helmet="true" rel="alternate" href="https://www.bbc.com/news/articles/c6v11qzyv8po" hrefLang="en"/>
> ```

@rhell the ticket description required that the canonical url should always be `.com`

**Comment by jimjohnsonrollings** [comment]
> Should this not be `.co.uk`?
> 
> ```
> <link data-react-helmet="true" rel="alternate" href="https://www.bbc.com/news/articles/c6v11qzyv8po" hrefLang="en"/>
> ```

Original issue speccing alternates is https://github.com/bbc/simorgh/issues/1338
AFAIK it's still correct to have all three alternates, all the time (for /news)

File: src/app/containers/StoryPromo/__snapshots__/index.test.jsx.snap [hunk 2]
```diff
@@ -376,22 +369,15 @@ exports[`StoryPromo Container should render correctly for canonical 1`] = `
...
+    font-size: 1.125rem;
+    line-height: 1.375rem;
   }
 }
 
 @media (min-width:37.5rem) {
   .c5 {
-    font-size: 1.125rem;
-    line-height: 1.375rem;
+    font-size: 1.25rem;
```
Comment (sareh) [inline src/app/containers/StoryPromo/__snapshots__/index.test.jsx.snap]: If you could run `npm run updateSnapshots` - these snapshots should not change any longer.

File: src/app/containers/Metadata/index.jsx [hunk 3]
```diff
@@ -44,10 +44,10 @@ const MetadataContainer = ({ metadata, promo }) => {
   const timeFirstPublished = new Date(metadata.firstPublished).toISOString();
   const timeLastPublished = new Date(metadata.lastPublished).toISOString();
 
-  const canonicalLink = `${origin}/${service}/articles/${id}`;
+  const canonicalLink = `https://www.bbc.com/${service}/articles/${id}`;
   const canonicalLinkUK = `https://www.bbc.co.uk/${service}/articles/${id}`;
   const canonicalLinkNonUK = `https://www.bbc.com/${service}/articles/${id}`;
-  const ampLink = `${origin}/${service}/articles/${id}.amp`;
+  const ampLink = `https://www.bbc.com/${service}/articles/${id}.amp`;
```
Comment (sareh) [inline src/app/containers/Metadata/index.jsx]: I just checked with @jimjohnsonrollings and the ampLink, which is used for the link: `<link rel="amphtml" href="" />` should vary depending on the origin of the user. 
```suggestion
  const ampLink = `${origin}/${service}/articles/${id}.amp`;
```
Only the canonical link should change.

**Review by sareh — COMMENTED** [review]
Looks great locally. Just added a couple of final comments below.

File: src/app/containers/Metadata/index.jsx [hunk 4]
```diff
@@ -20,7 +20,7 @@ const allTags = tags => {
 };
 
 const MetadataContainer = ({ metadata, promo }) => {
-  const { origin, platform } = useContext(RequestContext);
+  const { platform } = useContext(RequestContext);
```
Comment (sareh) [inline src/app/containers/Metadata/index.jsx]:
```suggestion
  const { origin, platform } = useContext(RequestContext);
```

**Review by 12 — APPROVED** [review]
Looks good!

**Review by dr3 — APPROVED** [review]

**Comment by jamesbrumpton** [comment]
LGTM.

**Pull request merged**